data/virt_autotest/setup_dns_service.txt

```diff
@@ -0,0 +1,252 @@
+#!/bin/bash
+
```

Comment:
Suggest to name the file as setup_dns_service.sh

Reply:
Done